shuttle/templates/schedules/shuttle_schedule.html

```diff
@@ -8,19 +8,17 @@
         <div class="center-page" style="padding-bottom: 1em">
             <div class="small-card">
                 <div class="card-body">
-                    <h6>Use <a href="https://docs.google.com/spreadsheets/d/1VGuZm58MfKW7TBFG09oGT8eIEAH1A4iFUg1utj1AA4s/edit?usp=sharing">this link </a>to change the schedule</h6>
-                    <h6>Please enter locations into the first row</h6>
+                    <h5>Use <a href="https://docs.google.com/spreadsheets/d/1VGuZm58MfKW7TBFG09oGT8eIEAH1A4iFUg1utj1AA4s/edit?usp=sharing">this link</a> to update the schedule</h5>
+                    <h6>Please insert the locations in the first row. Each column should have the</h6>
+                    <h6>departure times for that location; increasing the time as the table descends.</h6>
                     <h6>Times should be in the format (00:00 / 0:00) or (-) if the location has no time</h6>
                     <div class="center-page">
-                        <input id="submit-schedule-button" class="btn btn-warning btn-sm" type="submit" value="Submit to Database">
+                        <input id="submit-schedule-button" class="btn btn-warning btn-sm" type="submit" value="Sync Schedule With Spreadsheet">
```

Comment:
Could you possibly move the instructions and other administrative tools to the right of the spreadsheet so it would look more like this.

![Screen Shot 2020-08-03 at 2 54 05 PM](https://user-images.githubusercontent.com/26125143/89221635-35599f80-d599-11ea-857e-6ac60d35eebc.png)


Also add some more basic instructions as if you had no clue how to use it and or possibly even spreadsheet. I'm assuming that certain columns might always have to be some (ie locations have to be row 1 columns A - whatever it ends at) so it would probably be good to state this in the instructions as well.